com.vesc_project.VescTool.yaml

```diff
@@ -23,6 +23,7 @@ modules:
     buildsystem: simple
     build-commands:
       - install -D -m0644 com.vesc_project.VescTool.png /app/share/app-info/icons/flatpak/128x128/com.vesc_project.VescTool.png
```

Comment:
I don't think you should install anything in there.

Reply:
How should it be done?